apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -720,6 +734,22 @@ export class WorkshopForm extends React.Component {
     return value;
   };
 
+  // Selects the given value in the topic dropdown
+  handleSelect = event => {
+    const value = Object.keys(event)[0];
+    const isChecked = event[Object.keys(event)[0]];
+
+    let updatedTopics;
+    if (isChecked) {
+      // Add checked item into applied filters
+      updatedTopics = [...this.state.plTopics, value];
+    } else {
+      // Remove unchecked item from applied filters
+      updatedTopics = this.state.plTopics.filter(item => item !== value);
```

Comment:
does `filter` copy the list? if not, I think you'll need a spread here too

Reply:
Because we're starting with the plTopics list, filter is just removing anything that should be unchecked. But maybe I'm misunderstanding your comment?

Reply:
Got myself all turned around with immutability. React state should be immutable, but `filter` creates [a shallow copy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) of the array so I think you're good!